packages/features/bookings/components/FiltersContainer.tsx

```diff
@@ -1,51 +1,135 @@
+  const { data: query, pushItemToKey, removeItemByKeyAndValue, removeByKey } = useFilterQuery();
+  const [searchText, setSearchText] = useState("");
 
-type Filter = {
-  name: FilterTypes;
-  controllingQueryParams?: string[]; // this is what the filter controls - but also we show the filter if any of these query params are present
-  component: ReactNode;
-  showByDefault?: boolean;
-};
+  const members = trpc.viewer.teams.listMembers.useQuery({});
+
+  const filteredMembers = members?.data
+    ?.filter((member) => member.accepted)
+    ?.filter((member) =>
+      searchText.trim() !== ""
+        ? member?.name?.toLowerCase()?.includes(searchText.toLowerCase()) ||
+          member?.username?.toLowerCase()?.includes(searchText.toLowerCase())
+        : true
+    );
 
-const filters: Filter[] = [
-  {
-    name: "teams",
-    component: <TeamsMemberFilter />,
-    controllingQueryParams: ["teamId"],
-    showByDefault: true,
-  },
-  {
-    name: "people",
-    component: <PeopleFilter />,
-    controllingQueryParams: ["usersId"],
-    showByDefault: true,
-  },
-  {
-    name: "eventType",
-    component: <EventTypeFilter />,
-    controllingQueryParams: ["eventTypeId"],
-    showByDefault: true,
-  },
-];
+  const getTextForPopover = () => {
+    const userIds = query.userIds;
+    if (userIds) {
+      return `${t("people")}:  ${t("number_selected", { count: userIds.length })}`;
+    }
+    return `${t("people")}: ${t("all")}`;
+  };
+
+  return (
+    <AnimatedPopover text={getTextForPopover()}>
+      <FilterSearchField onChange={(e) => setSearchText(e.target.value)} placeholder={t("search")} />
+      <FilterCheckboxFieldsContainer>
+        {filteredMembers?.map((member) => (
+          <FilterCheckboxField
+            key={member.id}
+            id={member.id.toString()}
+            label={member?.name ?? member.username ?? t("no_name")}
+            checked={!!query.userIds?.includes(member.id)}
+            onChange={(e) => {
+              if (e.target.checked) {
+                pushItemToKey("userIds", member.id);
+              } else if (!e.target.checked) {
+                removeItemByKeyAndValue("userIds", member.id);
+              }
+            }}
+            icon={
+              <Avatar
+                alt={`${member?.id} avatar`}
+                imageSrc={member.username ? `${WEBAPP_URL}/${member.username}/avatar.png` : undefined}
+                size="xs"
+              />
+            }
+          />
+        ))}
+        {filteredMembers?.length === 0 && (
+          <h2 className="text-default px-4 py-2 text-sm font-medium">{t("no_options_available")}</h2>
+        )}
+      </FilterCheckboxFieldsContainer>
+    </AnimatedPopover>
+  );
+};
 
 export function FiltersContainer() {
+  const { t } = useLocale();
+
+  const [addFilterOptions, toggleOption, isFilterActive] = useBookingMultiFilterStore((state) => [
+    state.addFilterOptions,
+    state.toggleOption,
+    state.isFilterActive,
+    shallow,
```

Comment:
I don't think we are passing it correctly. It should have been an argument to `useBookingMultiFilterStore`